### PR TITLE
latest change for marketplace launch

### PR DESCRIPTION
**Pull request by netwmr01**
latest change for marketplace launch
addressed some comments from last pull request.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@netwmr01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: cloudera-on-centos/README.md [hunk 1]
```diff
@@ -1,14 +1,100 @@
...
+
+
+
+# Readme
+This template creates a multi-server Cloudera CDH 5.4.x Apache Hadoop deployment on CentOS virtual machines, and configures the CDH installation for either POC or high availability production cluster.
+
 The template also provisions storage accounts, virtual network, availability set, network interfaces, VMs, disks and other infrastructure and runtime resources required by the installation.
 
+The template expects the following parameters:
```
Comment (singhkay) [inline cloudera-on-centos/README.md]: This table of parameters should be removed

File: cloudera-on-centos/README.md [hunk 2]
```diff
@@ -1,14 +1,100 @@
...
+| virtualNetworkName | The name of the virtual network provisioned for the deployment | clouderaVnet |
+| subnetName | Subnet name for the virtual network where resources will be provisioned | clouderaSubnet |
+| addressPrefix | The network address space for the virtual network | 10.0.0.0/24 |
+| subnetPrefix | The network address space for the virtual subnet | 10.0.0.0/24 |
+| nodeAddressPrefix | The IP address prefix that will be used for constructing private IP address for each node in the cluster | 10.0.0. |
+| tshirtSize | T-shirt size of the Cloudera cluster (Eval, Prod) | Eval |
+| vmSize | The size of the VMs deployed in the cluster (Defaults to Standard_DS14) | Standard_DS14 |
+
+
 Topology
```
Comment (singhkay) [inline cloudera-on-centos/README.md]: This seems more important to be read than the EULA. Can you put this above the EULA?

File: cloudera-on-centos/azuredeploy.json [hunk 3]
```diff
@@ -11,73 +11,65 @@
     },
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: Please remove the default value for username

File: cloudera-on-centos/azuredeploy.json [hunk 4]
```diff
@@ -11,73 +11,65 @@
...
     "adminPassword": {
       "type": "securestring",
-      "defaultValue": "Eur32#1e",
       "metadata": {
         "description": "Admin password for the Virtual Machines"
       }
     },
+    "cmUsername": {
+      "type": "string",
+      "defaultValue": "cmadmin",
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: Please remove the default value for username

File: cloudera-on-centos/azuredeploy.json [hunk 5]
```diff
@@ -11,73 +11,65 @@
...
+    "cmPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "password for the Cloudera Manager"
+      }
+    },
     "storageAccountPrefix": {
       "type": "string",
-      "defaultValue": "defaultStorageAccountPrefix",
+      "defaultValue": "storageaccountprefix",
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: this needs to be globally unique. Please remove the default value

File: cloudera-on-centos/azuredeploy.json [hunk 6]
```diff
@@ -11,73 +11,65 @@
...
       "metadata": {
         "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed"
       }
     },
     "dnsNamePrefix": {
       "type": "string",
-      "defaultValue": "defaultDnsNamePrefix",
+      "defaultValue": "dnsnameprefix",
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: this needs to be globally unique. Please remove the default value

File: cloudera-on-centos/azuredeploy.json [hunk 7]
```diff
@@ -11,73 +11,65 @@
...
       "metadata": {
         "description": "Unique public dns name where the Virtual Machines will be exposed"
       }
     },
     "region": {
       "type": "string",
+      "defaultValue": "japanwest",
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: let the user pick a location and give a set of allowed values

**Pull request merged**